decentraland/unity-renderer
# style: fix typos on my profile section

## What does this PR change?

This PR was created to fix the issue [#5964 ](https://github.com/decentraland/unity-renderer/issues/5964)

## How to test the changes?

1- Open this [link](https://play.decentraland.org/?explorer-branch=style/ProfileTypos).
2- Open the main menu pressing tab. Then click the avatar+name button on the right top corner to open the dropdown menu. Afterwards, click on the `Account Settings` button.
3- In `My Profile` section, check that the typos pointed out in the issue #5964 are fixed.
